fortinet-fortirecon-aci/info.json

```diff
@@ -2277,95 +1717,1497 @@
         "nvd_v2_severity": "",
         "osint": [],
         "osint_hits": "",
-        "products": "",
+        "products": [],
         "social": [],
         "social_hits": "",
-        "vendors": ""
+        "vendors": []
       }
     },
     {
-      "operation": "get_vulnerability_intelligence_information",
-      "title": "Get Vulnerability Intelligence Information",
-      "annotation": "get_vulnerability_intelligence_information",
-      "description": "Retrieves vulnerability intelligence information for given organization ID",
+      "operation": "get_vulnerability_intelligence_vulnerable_products",
+      "title": "Get Vulnerability Intelligence vulnerable products",
+      "annotation": "get_vulnerability_intelligence_vulnerable_products",
+      "description": "Retrieves a list vulnerable products for the given organization ID and the input parameters specified.",
       "category": "investigation",
       "is_config_required": true,
       "visible": true,
       "enabled": true,
       "parameters": [
         {
-          "name": "fetch_info_for",
-          "title": "Fetch Information For",
+          "name": "input_src_type",
+          "title": "Input Source Type",
           "type": "select",
           "editable": true,
           "visible": true,
           "required": true,
-          "tooltip": "Select the type of information to fetch for vulnerability intelligence.",
-          "description": "Select the type of information to fetch for vulnerability intelligence.",
+          "tooltip": "Select the input source type based on which you want to filter the result. You can select between client and aci.",
+          "description": "Select the input source type based on which you want to filter the result. You can select between client and aci.",
           "options": [
-            "Global Notable CVEs Summary",
-            "Global Top Vendors By CVEs",
-            "Monitored CVEs Summary",
-            "Top Vendors By CVEs"
+            "client",
+            "aci"
           ]
-        }
-      ],
-      "conditional_output_schema": [
+        },
         {
-          "condition": "{{fetch_info_for === 'Global Notable'}}",
-          "output_schema": {
-            "critical": "",
-            "easm_added_cves": "",
-            "high": "",
-            "low": "",
-            "manual_added_cves": "",
-            "medium": "",
-            "total": ""
-          }
+          "name": "sort",
+          "title": "Sort By Product Count",
+          "type": "select",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Select the sort by type based on which you want to sort the result. You can select between count:desc and count:asc.",
+          "description": "Select the sort by type based on which you want to sort the result. You can select between count:desc and count:asc.",
+          "options": [
+            "count:desc",
```

Comment:
Can we change the option values to Accending and Decending

Reply:
Done